kubetest/kubeadmdind/kubeadm_dind.go

```diff
@@ -113,7 +114,7 @@ func NewDeployer(control *process.Control) (*Deployer, error) {
 		// Valid value
 	case "ipv6", "dual-stack":
 		log.Printf("Enabling IPv6")
-		if err := d.run("sysctl -w net.ipv6.conf.all.disable_ipv6=0"); err != nil {
```

Comment:
:/ how did this get through?

Reply:
that somehow works in the kubekins container :-)

Reply:
we run dind containers with elevated privileges

Reply:
does bazel still require privilege containers?

Reply:
no, but dind does

Reply:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml#L23-L25

we set privilege for all our bazel jobs though :-/ maybe only for jobs that needs build?

Reply:
Bazel jobs used to, they shouldn't anymore.

Reply:
Somewhat orthogonal though, in general jobs have a different environment from local so this stuff has to be caught by review.

Reply:
Right. Enabling IPv6 in the kubekins container is fine (doesn't effect the host), but setting it locally when running UT is not good. I should have caught this when I moved the sysctl and added UT that exercised this. I can't think of any practical way to detect this in UT. Thanks for fixing this.